src/js/dateTime.mjs

```diff
@@ -0,0 +1,19 @@
+const displayCurrentTime = () => {
+    const currentTimeElement = document.getElementById("clock");
+  
+    const now = new Date();
+  
+    const dayOfWeek = now.toLocaleDateString("en-US", { weekday: "short" });
+    const day = now.getDate();
+    const month = now.toLocaleDateString("en-US", { month: "short" });
+    const hours = now.getHours();
+    const minutes = now.getMinutes();
+  
+    const formattedTime = `${dayOfWeek} ${day} ${month} ${hours}:${minutes
+      .toString()
+      .padStart(2, "0")}`;
+  
+    currentTimeElement.textContent = formattedTime;
```

Comment:
It might be more reusable if this function just returned a time string and you had another function for updating the dom. Small change though